src/index.js

```diff
@@ -27,6 +28,10 @@ const injectDOM = function CozyBarInjectDOM ({lang, appName, iconPath}) {
   })
 }
 
+const getDefaultStackURL = function GetDefaultCozyURL () {
+  return document.querySelector('[role=application]').dataset.cozyDomain
```

Comment:
Imagine that some contributors try the cozy-bar for the first time but did not start their application with our template and do not have a `[role=application]` element. Shouldn't we provide et better experience by throwing a clear error like "Your application need an element having with a role=application attribute" ? Here it will just crash.